include/boost/crc.hpp

```diff
@@ -1774,7 +1774,14 @@ crc_basic<Bits>::process_bits
     unsigned char const  high_bit_mask = 1u << ( CHAR_BIT - 1u );
     for ( std::size_t i = bit_length ; i > 0u ; --i, bits <<= 1u )
     {
+#ifdef BOOST_MSVC
+#pragma warning(push)
+#pragma warning(disable:4800)
+#endif
         process_bit( static_cast<bool>(bits & high_bit_mask) );
```

Comment:
`(bits & high_bit_mask) > 0u` is a much better fix for this warning

Reply:
It would also be nice to actually enable the CI providers for this repository.

Reply:
Thanks @Kojoley, adjusted accordingly.